app/components/UI/FiatOnRampAggregator/Views/GetQuotes.tsx

```diff
@@ -328,69 +328,76 @@ const GetQuotes = () => {
       !isFetchingQuotes &&
       pollingCyclesLeft >= 0
     ) {
-      const quotesWihoutError: QuoteResponse[] = quotes
+      const quotesWithoutError: QuoteResponse[] = quotes
         .filter(({ error }) => !error)
```

Comment:
I suggest we use Boolean for dataset filtering. 
```suggestion
        .filter(({ error }) => !Boolean(error))
```

Reply:
! operator already casts to boolean, this is not necessary.

Reply:
Resolved!